src/utils/hints/namesHints.js

```diff
@@ -1,5 +1,5 @@
 export const namesHints = {
-  name: 'Unique identifier that may be configured to point to accounts, smart contracts or oracles.',
+  name: 'A unique identifier that may be configured to point to accounts, smart contracts or oracles.',
```

Comment:
Fair point. I hope the second iteration makes more sense. 

I'm trying to limit the hints because on mobile it will take significant amount of space to display the message, while there is a "Learn more" button with infinitely more detailed description